docs/validator/compile-and-run-a-node.md

```diff
@@ -199,15 +199,25 @@ $ ./target/release/neard --home ~/.near init --chain-id testnet --download-genes
 > You can skip the `--home` argument if you are fine with the default working directory in `~/.near`. If not, pass your preferred location.
 
 This command will create the required directory structure and will generate `config.json`, `node_key.json`, and `genesis.json` files for `testnet` network.
-- `config.json` - Configuration parameters which are responsive for how the node will work. This file should contain the following fields critical for validator nodes:
-  - `"tracked_shards": []` - to enable single-shard tracking. The shard the validator should track will be assigned by the protocol.
```

Comment:
are we removing it since they are already taken care of in configs?

Reply:
Yes